consensus/spos/worker.go

```diff
@@ -485,7 +499,8 @@ func (wrk *Worker) shouldBlacklistPeer(err error) bool {
 		errors.Is(err, errorsErd.ErrSignatureMismatch) ||
 		errors.Is(err, nodesCoordinator.ErrEpochNodesConfigDoesNotExist) ||
 		errors.Is(err, ErrMessageTypeLimitReached) ||
-		errors.Is(err, ErrEquivalentMessageAlreadyReceived) {
+		errors.Is(err, ErrEquivalentMessageAlreadyReceived) ||
```

Comment:
also here, maybe epoch activation check. If some malicious node sends EquivalentMessages or InvalidSigners before the activation, it might blacklist the entire network.

Reply:
added

Reply:
is it added somewhere else?

Reply:
ok